src/contributors/contributors.service.ts

```diff
@@ -15,6 +15,12 @@ export class ContributorsService {
     return this.contributorsRepository.find();
   }
 
+  findOne(username: string): Promise<Contributor> {
```

Comment:
```suggestion
  findOneByUsername(username: string): Promise<Contributor> {
```

Renaming the method "findOne" to "findOneByUsername" helps the developer understand how the find operate and what information they need to pass in.